### PR TITLE
Pr 56eval qspiflash

### PR DESCRIPTION
**Pull request by gkalsi**

### CODE REVIEW

**Comment by travisg** [comment]
Can you flatten these two commits and re-push request? One fixes the other for at least some things.

File: lib/bio/debug.c [hunk 1]
```diff
@@ -288,7 +290,7 @@ static int cmd_bio(int argc, const cmd_args *argv)
 static bool is_valid_block(bdev_t *device, bnum_t block_num, uint8_t* pattern,
                            size_t pattern_length)
 {
-    uint8_t *block_contents = malloc(device->block_size);
+    uint8_t *block_contents = memalign(DMA_ALIGNMENT, device->block_size);
```
Comment (travisg) [inline lib/bio/debug.c]: a test comment on the unified view.

File: lib/bio/debug.c [hunk 2]
```diff
@@ -331,13 +333,19 @@ static ssize_t erase_test(bdev_t *device)
     return num_invalid_blocks;
 }
 
+uint8_t get_signature(uint32_t word)
```
Comment (travisg) [inline lib/bio/debug.c]: static

**Comment by gkalsi** [comment]
Merged.

**Pull request closed without merging**